### PR TITLE
test: New 2.5 tests + links from test specs to 2.5 test plan

### PR DESCRIPTION
**Pull request by MaximKhlobystov**
This PR includes new test specs for presentation and connection status test suites:
- multi-presentation upload
- fit to width
- copy connection stats
- use link to settings when there's a connection issue.

It also adds test plan links to the test specs.

### CODE REVIEW

File: bigbluebutton-tests/playwright/connectionStatus/connectionStatus.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { MultiUsers } = require('../user/multiusers');
 const e = require('../core/elements');
 const { ELEMENT_WAIT_TIME } = require('../core/constants');
 const { openConnectionStatus, checkNetworkStatus } = require('./util');
-
+const { sleep } = require('../core/helpers');;
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/connectionStatus/connectionStatus.js:6]: duplicated `;`

File: bigbluebutton-tests/playwright/connectionStatus/connectionStatus.js [hunk 2]
```diff
@@ -38,6 +38,24 @@ class ConnectionStatus extends MultiUsers {
...
+    await this.modPage.waitForSelector(e.dataSavingsTab);
+  }
+
+  async copyStatsTest(context) {
+    await openConnectionStatus(this.modPage);
+    await this.modPage.waitAndClick(e.copyStats);
+    await context.grantPermissions(['clipboard-write', 'clipboard-read'], { origin: process.env.BBB_URL });
+    const copiedText = await this.modPage.page.evaluate(async () => navigator.clipboard.readText());
+    const check = copiedText.includes("audioCurrentUploadRate");
+    expect(check).toBeTruthy();
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/connectionStatus/connectionStatus.js:57]: missing `await`

**Review by antonbsa — COMMENTED** [review]
overall, very good additions! I'm leaving just a few comments and suggestions below:

**Comment by BrentBaccala** [comment]
I can't figure how to use the github system to propose a change to a file that's not already in the PR, so I'm not really sure how to propose this change. (Maybe somebody can teach me.)

Anyway, some changes are needed in `polling/poll.js`.  Grep in that file for `uploadPresentation`; it appears twice.  These need to be changed to `uploadSinglePresentation`.

**Comment by MaximKhlobystov** [comment]
> I can't figure how to use the github system to propose a change to a file that's not already in the PR, so I'm not really sure how to propose this change. (Maybe somebody can teach me.)
> 
> Anyway, some changes are needed in `polling/poll.js`. Grep in that file for `uploadPresentation`; it appears twice. These need to be changed to `uploadSinglePresentation`.

Good catch! I've pushed the updates. There's a another place that required similar change: `notifications/presenterNotifications.js`.

**Review by antonbsa — APPROVED** [review]

**Pull request merged**